### DISCUSSION TITLE
Next release v2.12.0 or v2.11.1 date?

### DISCUSSION THREAD

**Discussion by jools-uk** [jxlsteam/jxls#183] [Q&A] [answered]
Hi, do you have a date for the next release (or at minimum an RC)?
I have a restriction that I can only use external packages if they are pulled from Maven Central.
Thanks!

**Comment by SoltauFintel** [comment, marked as answer]
jxls is primarily a hobby project. There is therefore generally no release planning.

bug #173 needs to be fixed first

and there are more open tickets for 2.12.0: https://github.com/jxlsteam/jxls/milestone/6

**Reply by SoltauFintel** [reply]
We have now managed to fix #173. 😃 

What's next?
I'll do a full test run again.
More issues are still to be processed. https://github.com/jxlsteam/jxls/milestone/6
I can well imagine that 2.12.0 will be released in May.

**Reply by SoltauFintel** [reply]
My issues are done. I can start the RC test. Maybe we can release at the weekend.

**Reply by SoltauFintel** [reply]
RC test done. Releasing the next days.

**Reply by SoltauFintel** [reply]
V2.12.0 has been released.
